tsdef/twilio-video-tests.ts

```diff
@@ -19,6 +19,9 @@ function getAudioTrack(track: Video.LocalAudioTrack) {
   const localAudioTrack = track;
   localAudioTrack.attach();
   localAudioTrack.detach();
+  localAudioTrack.disable();
+  localAudioTrack.enable();
+  localAudioTrack.stop();
```

Comment:
@PikaJoyce Please store the return value in a variable of type `LocalAudioTrack` (`LocalVideoTrack` in video tests). The objective is to make sure we are returning the proper video types.